src/main/java/com/nylas/NeuralSentimentAnalysis.java

```diff
@@ -0,0 +1,55 @@
+package com.nylas;
+
+public class NeuralSentimentAnalysis {
+	private String account_id;
+	private String sentiment;
+	private float sentiment_score;
+	private int processed_length;
+	private String text;
+
+	public String getAccountId() {
+		return account_id;
+	}
+
+	public void setAccountId(String accountId) {
+		this.account_id = accountId;
+	}
```

Comment:
I thought about doing that actually and for some reason I didn't, I'll remove setters from all the Neural classes since all the endpoints are read-only.